sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_models.py

```diff
@@ -290,10 +281,9 @@ class TransactionReceipt(object):
 
     def __init__(
         self,
-        transaction_id,  # type: str
-        receipt,  # type: List[int]
-    ):
-        # type: (int, List[int]) -> None
+        transaction_id: str,
+        receipt: List[int],
+    ) -> None:
```

Comment:
if you have time, it would be nice to update all the type hints from comment -> annotation style

Reply:
Yes, I did think about that, but don't have the bandwidth for it at the moment 😞 It is a bit much because it is not only this `_models.py` file but all the non-async files too (e.g. client, etc).

I think we will get to it when we are ready for GA.